.scripts/remove_duplicates.py

```diff
@@ -30,59 +37,43 @@ def remove_duplicates(original_csv, duplicate_statements, translated_statements)
-        print(f"Cleaned original file has been saved to {translated_output_file}")
+    if indices_to_remove:
+        # remove the duplicate indices from the translated CSVs
+        for translated_csv in translated_statements:
+            translated_df = pd.read_csv(translated_csv)
+            translated_df = translated_df.drop(list(indices_to_remove))
+            translated_df.reset_index(drop=True)
+            translated_output_file = f'raw_statements/{os.path.basename(translated_csv)}'
+            translated_df.to_csv(translated_output_file, index=False)
+            print(f"Cleaned translated file has been saved to {translated_output_file}")
+            print(f"removed {len(indices_to_remove)} duplicates")
+            print('-----------------')
+
+        # drop duplicate indices from the original CSV
+        original_df = original_df.drop(list(indices_to_remove))
+        original_df.reset_index(drop=True)
+        original_output_file = f'raw_statements/{os.path.basename(original_csv)}'
+        original_df.to_csv(original_output_file, index=False)
+        print(f"Cleaned original file has been saved to {original_output_file}")
         print(f"removed {len(indices_to_remove)} duplicates")
         print('-----------------')
 
-    # drop duplicate indices from the original CSV
-    original_df = original_df.drop(list(indices_to_remove))
-    original_df.reset_index(drop=True)
-    original_output_file = 'raw_statements_cleaned/' + os.path.basename(original_csv)
-    original_df.to_csv(original_output_file, index=False)
-    print(f"Cleaned original file has been saved to {original_output_file}")
-    print(f"removed {len(indices_to_remove)} duplicates")
-    print('-----------------')
-
-# function that tests the number of lines across files is the same
-def test_line_numbers(original_csv, translated_statements):
-    original_df = pd.read_csv(original_csv)
-    original_lines = len(original_df)
-
-    for translated_csv in translated_statements:
-        translated_df = pd.read_csv(translated_csv)
-        translated_lines = len(translated_df)
-        if (translated_lines != original_lines):
-            print(f"Error: {translated_csv} has a different number of lines compared to {original_csv}")
-            return False
-    print("All files have the same number of lines")
-    return True
-
 if __name__ == "__main__":
-    languages = ['ar','bn','es','fr','hi','ja','pt','ru','zh']
-
-    observable_translated_statements = [f'raw_statements/observable_gpt4o_{language}.csv' for language in languages]
-    observable_duplicate_statements = [f'duplicate_statements/observable_gpt4o_{language}_duplicates.csv' for language in languages]
-
-    email_translated_statements = [f'raw_statements/email_statements_{language}.csv' for language in languages]
-    email_duplicate_statements = [f'duplicate_statements/email_statements_{language}_duplicates.csv' for language in languages]
+    if len(sys.argv) < 2:
```

Comment:
This isn't a comment to be resolved just a recommendation, I use `argparse` for parsing args from command line. It has -h help and some other nice things baked in.

Reply:
Ok, I changed it to use argparse. Will keep this in mind in the future. Thanks!